irr/src/CMakeLists.txt

```diff
@@ -228,7 +247,7 @@ set(link_includes
 	"${ZLIB_INCLUDE_DIR}"
```

Comment:
above here is the check for `CHECK_GL_VERSION_4_5`. it now doesn't work anymore.
I think it can just be skipped for SDL3, since we know that it has a new enough header?